restcomm/restcomm.dao/src/main/java/org/mobicents/servlet/restcomm/entities/Sid.java

```diff
@@ -34,7 +34,7 @@
     private final String id;
 
     public enum Type {
-        ACCOUNT, APPLICATION, ANNOUNCEMENT, CALL, CLIENT, CONFERENCE, GATEWAY, INVALID, NOTIFICATION, PHONE_NUMBER, RECORDING, REGISTRATION, SHORT_CODE, SMS_MESSAGE, TRANSCRIPTION, INSTANCE
+        ACCOUNT, APPLICATION, ANNOUNCEMENT, CALL, CLIENT, CONFERENCE, GATEWAY, INVALID, NOTIFICATION, PHONE_NUMBER, RECORDING, REGISTRATION, SHORT_CODE, SMS_MESSAGE, TRANSCRIPTION, INSTANCE, GEOLOCATION
```

Comment:
Sorry @jaimecasero, I didn't get this one (?).
Anyway, I just added the Geolocation code in this class (GL).